src/WordPress/Hooks.php

```diff
@@ -322,4 +322,36 @@ public function initAutomaticPlatformOptimization()
             header('cf-edge-cache: no-cache');
         }
     }
+
+    public function purgeCacheOnCommentStatusChange($new_status, $old_status, $comment)
+    {
+        if (!is_object($comment)) {
+            return; // nothing to do
+        }
+        if (empty($comment->comment_post_ID)) {
+            return; // nothing to do
+        }
```

Comment:
we can opt to simplify this one a little with `!isset` and `empty` checks in the same expression

```suggestion
        if (!isset($comment->comment_post_ID) || empty($comment->comment_post_ID)) {
            return;  
        }
```

Reply:
That is right, Jacob - thank you.